src/components/Roadmap/index.tsx

```diff
@@ -12,7 +13,7 @@ const techCategories: TechCategory[] = [
     id: 1,
     title: "Frontend",
     technologies: [
-      { id: 1, name: "HTML", link: "#" },
+      { id: 1, name: "HTML", link: "HtmlRoadMap" },
```

Comment:
replace `HtmlRoadMap` to `/roadmap/html`

Reply:
resolved